crates/ide_assists/src/handlers/replace_if_let_with_match.rs

```diff
@@ -44,6 +44,14 @@ use crate::{
 // ```
 pub(crate) fn replace_if_let_with_match(acc: &mut Assists, ctx: &AssistContext) -> Option<()> {
     let if_expr: ast::IfExpr = ctx.find_node_at_offset()?;
+    let available_range = TextRange::new(
+        if_expr.syntax().text_range().start(),
+        if_expr.then_branch()?.syntax().text_range().start(),
```

Comment:
> Instead, it should only be avaialbe between if and {.

I'm not sure if the bounds should be included but since origin tests contains `$0if` so i make the range to be`if` .. `before {`.

To include the end of `{`, maybe change the end to `.start() + 1`?